layouts/partials/scripts.html

```diff
@@ -6,13 +6,15 @@
 <script src="//cdnjs.cloudflare.com/ajax/libs/waypoints/4.0.1/jquery.waypoints.min.js"></script>
 <script src="//cdnjs.cloudflare.com/ajax/libs/Counter-Up/1.0/jquery.counterup.min.js"></script>
 <script src="//cdnjs.cloudflare.com/ajax/libs/jquery-parallax/1.1.3/jquery-parallax.js"></script>
+{{ if .Site.Params.enableGoogleMaps }}
```

Comment:
I would take care about people that already have the theme and doesn't have this option enabled.

Reply:
@ryanfox1985  We could simply mention it as a "breaking change" with the next [release](https://github.com/devcows/hugo-universal-theme/releases).

The only proper solution I can think of to make this "backwards compatible" would be to change this whole option to the opposite, i.e. `disableGoogleMaps` instead of `enableGoogleMaps`. But honestly I like the "positively phrased" `enableGoogleMaps` way better.

Reply:
ok